app/src/main/java/com/wednesday/banklocator/activity/MainActivity.kt

```diff
@@ -2,15 +2,23 @@ package com.wednesday.banklocator.activity
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.lifecycle.ViewModelProvider
 import androidx.navigation.fragment.findNavController
 import androidx.navigation.ui.setupWithNavController
 import com.wednesday.banklocator.R
+import com.wednesday.banklocator.repository.NewsRepository
+import com.wednesday.banklocator.viewmodel.BankViewModel
+import com.wednesday.banklocator.viewmodel.BankViewModelFactory
 import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
+    lateinit var viewModel: BankViewModel
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         bottomNavigationView.setupWithNavController(navHostFragment.findNavController())
+        val newsRepository = NewsRepository()
```

Comment:
Why is this object called NewsRepository?

Reply:
I'll change that.